### ISSUE TITLE
dotnet CLI

### ISSUE THREAD

**Issue by Reonekot** [daveaglick/Scripty#30]
Is there a way to get this working with the new RTM version of dotnet CLI and ASP.NET Core (MVC)? Or is the .xproj model not supporter?

**Comment by daveaglick** [comment]
No, not right now. Part of the issue is that while the .NET Core runtime is 1.0, the tooling is still in preview. A lot of the existing tooling like `project.json` is going to be reworked for the final version, and it's not totally clear what that's going to look like. While getting Scripty to run on .NET Core probably wouldn't be too hard, it's the integration with the tooling that's the blocker right now.

While Microsoft has committed to make the transition for projects based on the preview tooling to the release tooling transparent using automated porting, etc. that probably isn't going to be the case for something like Scripty that integrates so tightly. Instead of spending a lot of time working that out, I'm probably going to wait until the next version of Visual Studio gets closer and the tooling story gets clearer.

Note that I think you can still use Scripty in a .NET Core / ASP.NET Core application (though I haven't tried). It's a Visual Studio addon and/or MSBuild task, so as long as you're using Visual Studio and/or MSBuild, it should work. You just may not have access to all the project information you would get for a .NET Framework application.
